File: python/prism_sdk/local_knowledge_representation_workflow_fabric.py
"""Worldgen P04 AFA-worldgen-P04-F13 product wrapper."""
from .worldgen_knowledge_workflow_support import KnowledgeWorkflowRequest, KnowledgeWorkflowReceipt, manifest as _manifest, schedule as _run
FEATURE_ID="AFA-worldgen-P04-F13"
CONTRACT_VERSION="worldgen-local-knowledge-workflow/1.0"
def worldgen_local_knowledge_representation_workflow_fabric_manifest(): return _manifest(feature_id=FEATURE_ID, contract_version=CONTRACT_VERSION, input_schema="KnowledgeWorkflowRequest1@1", scale="local single-study", autonomy_tier="A1")
def schedule_worldgen_local_knowledge_representation_workflow(request): return _run(request, feature_id=FEATURE_ID, contract_version=CONTRACT_VERSION, scale="local single-study", require_approval=False, require_federation=False)
__all__=["KnowledgeWorkflowRequest","KnowledgeWorkflowReceipt","worldgen_local_knowledge_representation_workflow_fabric_manifest","schedule_worldgen_local_knowledge_representation_workflow"]

File: crates/worldgen/src/local_knowledge_representation_workflow_fabric.rs
//! Worldgen P04-F13 local knowledge-representation workflow fabric.
use super::knowledge_workflow_support::{self, KnowledgeWorkflowRequest, KnowledgeWorkflowReceipt};
pub const FEATURE_ID:&str="AFA-worldgen-P04-F13"; pub const CONTRACT_VERSION:&str="worldgen-local-knowledge-workflow/1.0";
pub fn worldgen_local_knowledge_representation_workflow_fabric_manifest()->serde_json::Value{knowledge_workflow_support::manifest(FEATURE_ID,CONTRACT_VERSION,"KnowledgeWorkflowRequest1@1","local single-study","A1")}
pub fn schedule_worldgen_local_knowledge_representation_workflow(request:&KnowledgeWorkflowRequest)->Result<KnowledgeWorkflowReceipt,knowledge_workflow_support::KnowledgeWorkflowError>{knowledge_workflow_support::schedule(request,FEATURE_ID,CONTRACT_VERSION,"local single-study",false,false)}
pub use knowledge_workflow_support::{KnowledgeWorkflowError,KnowledgeWorkflowReceipt as WorldgenLocalKnowledgeWorkflowReceipt,KnowledgeWorkflowRequest as WorldgenLocalKnowledgeWorkflowRequest};
